### PR TITLE
Adding "Age x Risk for Severe COVID-19" to the demographic table

### PR DESCRIPTION
**Pull request by cyu-hvtn**

### CODE REVIEW

File: cor_tabular/code/make_table_all.R [hunk 1]
```diff
@@ -396,7 +405,7 @@ tab_dm <- full_join(
               names_sort = T,
               values_from = c(rslt)) %>%
   mutate(Characteristics = factor(subgroup_cat,
-                                  levels=c("Age $\\geq$ 65","Age $<$ 65",
+                                  levels=c("Age $<$ 65", "Age $\\geq$ 65",
```
Comment (benkeser) [inline cor_tabular/code/make_table_all.R:408]: Is the change of ordering here correct?

File: cor_tabular/code/make_table_all.R [hunk 2]
```diff
@@ -410,16 +419,18 @@ tab_dm <- full_join(
                                            "Communities of Color",
                                            "Hispanic or Latino","Not Hispanic or Latino",
                                            "Not reported and unknown ",
-                                           "At-risk","Not at-risk"))) %>% 
+                                           "At-risk","Not at-risk",
+                                           "Age $<$ 65 At-risk","Age $<$ 65 Not at-risk",
```
Comment (benkeser) [inline cor_tabular/code/make_table_all.R:423]: Why not $<$ as you do for `$\\geq$`?

File: cor_tabular/code/make_table_all.R [hunk 3]
```diff
@@ -26,7 +26,7 @@ tlf <-
       the baseline SARS-CoV-2 negative per-protocol cohort",
       table_footer = "This table summarises the random subcohort, which
       was randomly sampled from the per-protocol individuals without a COVID failure
-      event < 7 days post Day 57. The sampling was stratified by the key baseline 
+      event $<$ 7 days post Day 57. The sampling was stratified by the key baseline
```
Comment (benkeser) [inline cor_tabular/code/make_table_all.R:29]: Is there a reason this is not `$<$`?

File: cor_tabular/code/make_table_all.R (hunk 1 above)
Comment (cyu-hvtn) [reply cor_tabular/code/make_table_all.R:408]: Yes, these are for the subgroup "Age" not "Age, Risk for Severe COVID". Should I remove "Age" as it became somehow redundant now?

File: cor_tabular/code/make_table_all.R (hunk 2 above)
Comment (cyu-hvtn) [reply cor_tabular/code/make_table_all.R:423]: Sorry not sure what the question means...

File: cor_tabular/code/make_table_all.R (hunk 3 above)
Comment (cyu-hvtn) [reply cor_tabular/code/make_table_all.R:29]: The change is from < to $<$

File: cor_tabular/code/make_table_all.R (hunk 1 above)
Comment (benkeser) [reply cor_tabular/code/make_table_all.R:408]: Whatever you think is better. Just wanted to make sure the ordering was correct!

File: cor_tabular/code/make_table_all.R (hunk 2 above)
Comment (benkeser) [reply cor_tabular/code/make_table_all.R:423]: Sorry for both questions, I am asking why you are not using Latex math expression `$<$` like you do for the greater than or equal to sign `$\\ge$`

File: cor_tabular/code/make_table_all.R (hunk 3 above)
Comment (benkeser) [reply cor_tabular/code/make_table_all.R:29]: Ok, it must just be rendering funny on my browser.

File: cor_tabular/code/make_table_all.R (hunk 1 above)
Comment (benkeser) [reply cor_tabular/code/make_table_all.R:408]: Let me know if you want to make another change; otherwise I will merge. 

Thanks!!

Comment (cyu-hvtn) [reply cor_tabular/code/make_table_all.R:408]: I think no further changes needed :) 
Thank you!

**Pull request merged**